eval/listing.elv

```diff
@@ -0,0 +1,53 @@
+fn location {
+    candidates = [(dirs | each {
+	score = (splits $args[0][score] &sep=. | take 1)
+        put [
+            &content=$args[0][path]
+            &display=$score" "$args[0][path]
```

Comment:
My thinking was to let user control the display setting while listing will in charge of drawing.